pkg/util/mount/mount_linux.go

```diff
@@ -185,12 +185,14 @@ func (mounter *Mounter) PathIsDevice(pathname string) (bool, error) {
 }
 
 func exclusiveOpenFailsOnDevice(pathname string) (bool, error) {
-	if isDevice, err := pathIsDevice(pathname); !isDevice {
+	// stat the path, return error if the path exists but is not a device
+	if isDevice, err := pathIsDevice(pathname); !isDevice && !os.IsNotExist(err) {
```

Comment:
I am not sure about the logic here. If path does not exist, it will move on to call syscall.Open which will return an error? And DeviceOpened will return an error too?